spec/models/bus_stops_route_spec.rb

```diff
@@ -35,4 +35,57 @@
       end
     end
   end
+  describe 'import' do
+    let(:main_variant) { %w[A B C D E] }
+    before :each do
+        @stop_hash = { route_dir: { key_1: main_variant, key_2: other_variant } }
+    end
+    let(:resequence!) { BusStopsRoute.import(@stop_hash) }
+    let(:other_variant) { [] }
```

Comment:
I don't think it's possible for the other variant to be truly empty, right?

Reply:
Agreed. I wanted the `main_variant` to be the only variant, and thus contain the most stops already in order.

Reply:
Okay. That's not quite what this accomplishes, though.

My thinking was that we wanted a scenario where there was another variant whose stops were a proper subset of the main variant, to make sure that the main variant wouldn't get reordered as a result.